app/models/goal.py

```diff
@@ -3,3 +3,20 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.Text)
+
+    tasks = db.relationship("Task", back_populates="goal", lazy=True)
```

Comment:
Great sync with the Task model & good job remembering the `lazy` loading parameter! 💯